docs/user/exercises/assessment.rst

```diff
@@ -0,0 +1,84 @@
+.. _assessment:
+
+Assessment
+==========
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+
+Overview
+--------
+
+Artemis offers three different modes of exercise assessment:
+
+- **Manual:** Tutors must manually grade the submissions of students.
+- **Automatic:** Artemis automatically grades the submissions of students.
+- **Semi-Automatic:** Artemis automatically grades the submissions of students, but the tutors manually grade all submissions afterward.
```

Comment:
Not sure about the exact formulation, but I think this sentence should be more clear. 
```suggestion
- **Semi-Automatic:** Artemis provides an automatic starting point for the tutors to manually improve the grading afterward.
```